wordpress-mobile/WordPress-iOS
# Decoupling Authentication Flow: Mark IV

### Description:
In this PR we're implementing the **WordPressAuthenticatorDelegate** protocol, which, at this stage is responsible of:

- Indicating if a `Support` button should be added at the top right corner of the Login UI.
- Providing a ViewController, to be presented whenever such button is pressed.

@nheagy (Whenever you've got a few spare cycles, next week!!!) would you mind reviewing this one?
Thanks in advance!!!


### Details:
**WordPressAuth.framework** will contain the `WordPressSupportSourceTag` enum, which is a subset of WPiOS's `SupportSourceTag` enum.

Reason: one will live at a framework level, another one will be client app level.

### Testing:
1. Launch WPiOS (fresh install)
2. Verify that the top right button (Help) works as expected, in both, Login and Signup flows.
